src/pages/ScoreReport.vue

```diff
@@ -771,25 +787,47 @@ function scoreFieldAboveSixth(taskId) {
   return 'percentile';
 }
 
+const descriptionsByTaskId = {
```

Comment:
Not essential, but this could be another candidate for being moved to the helper file. Just to reduce bloat in this file and make the text easily accessible for editing.